supervision/detection/core.py

```diff
@@ -187,10 +187,17 @@ def from_yolov8(cls, yolov8_results) -> Detections:
             >>> detections = sv.Detections.from_yolov8(result)
             ```
         """
+        masks = None
+        if yolov8_results.masks:
+            masks = yolov8_results.masks.masks.cpu().numpy()  # masks, (N, H, W)
+            masks = np.moveaxis(masks, 0, -1)  # masks, (H, W, N)
+            masks = cv2.resize(masks, (yolov8_results.masks.orig_shape[1], yolov8_results.masks.orig_shape[0]))
```

Comment:
When I run the code I get:

```
0: 640x384 1 person, 1 car, 1 dog, 1 backpack, 1 handbag, 153.3ms
Speed: 15.4ms preprocess, 153.3ms inference, 37.7ms postprocess per image at shape (1, 3, 640, 640)
WARNING ⚠️ 'Masks.masks' is deprecated. Use 'Masks.data' instead.
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
[<ipython-input-10-6c1542ea3727>](https://localhost:8080/#) in <cell line: 2>()
      1 results = model.predict(source=image)[0]
----> 2 detections = sv.Detections.from_yolov8(results)

[/content/supervision/supervision/detection/core.py](https://localhost:8080/#) in from_yolov8(cls, yolov8_results)
    192             masks = yolov8_results.masks.masks.cpu().numpy()  # masks, (N, H, W)
    193             masks = np.moveaxis(masks, 0, -1)  # masks, (H, W, N)
--> 194             masks = cv2.resize(masks, (yolov8_results.masks.orig_shape[1], yolov8_results.masks.orig_shape[0]))
    195             masks = np.moveaxis(masks, -1, 0)  # masks, (N, H, W)
    196         return cls(

NameError: name 'cv2' is not defined
```

Reply:
Let me test once again.

Reply:
I think something still does not work as expected. I've done some tests. Here are two images. First produced by YOLOv8 annotators and second by our annotators. Looks like masks are not scaled properly for some reason.  

- YOLOv8 result:

![2fb4d321-88a5-4f0b-b9b3-ced5c5fe12ee](https://github.com/roboflow/supervision/assets/26109316/6b43de14-e82c-4b16-9446-0c9c42a079cf)

- Supervision result:

![download - 2023-05-23T192244 934](https://github.com/roboflow/supervision/assets/26109316/6bc21e97-c874-476f-b48c-42374fa5c4d4)

Reply:
@SkalskiP it is due to letterbox. Padding is not zero in this case. We need an information of inference input size.

Reply:
@hardikdava is it possible to extract that info from `results`?

Reply:
@SkalskiP any update? I think we are ready to merge it.

Reply:
Hi, @hardikdava 👋🏻 ! There are a few formatting changes I'd do, but I don't want to bother you with those. I'll take care of it on my own. We are merging.